public/controllers/agent/wazuh-config/index.ts

```diff
@@ -0,0 +1,373 @@
+const architectureButtons = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+  {
+    id: 'armhf',
+    label: 'armhf',
+  },
+  {
+    id: 'aarch64',
+    label: 'aarch64',
+  },
+];
+
+const architectureButtonsWithPPC64LE = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+  {
+    id: 'armhf',
+    label: 'armhf',
+  },
+  {
+    id: 'aarch64',
+    label: 'aarch64',
+  },
+  {
+    id: 'powerpc',
+    label: 'powerpc',
+  },
+];
+
+
+const architectureButtonsi386 = [
+  {
+    id: 'i386',
+    label: 'i386',
+  }
+];
+
+const architecturei386Andx86_64 = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+];
+
+const architectureButtonsSolaris = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'sparc',
+    label: 'SPARC',
+  }
+];
+
+const architectureButtonsMacos = [
+  {
+    id: 'intel',
+    label: 'Intel'
+  },
+  {
+    id: 'applesilicon',
+    label: 'Apple Silicon'
+  }
+]
+
+const architectureButtonsOpenSuse = [
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+  {
+    id: 'ARM64',
+    label: 'ARM64',
+  }
+];
+
+const architectureButtonsAix = [
+  {
+    id: 'powerpc',
+    label: 'PowerPC',
```

Comment:
**nitpick**: Other architectures are displayed as `powerpc`. If they are the same, we should have consistency in the names.

![image](https://user-images.githubusercontent.com/34042064/197546654-b4c21869-3213-409c-b9c4-317431b33b85.png)

Reply:
Fixed